packages/world/src/Vec3.sol

```diff
@@ -218,6 +236,12 @@ library Vec3Lib {
     return chebyshevDistance(self, other) <= radius;
   }
 
+  function inSphere(Vec3 self, Vec3 other, uint256 radius) internal pure returns (bool) {
+    Vec3 d = other - self;
+    // TODO: should probably use direct subtraction of components to avoid overflow issues
```

Comment:
Seems like we should resolve this todo no? Otherwise we could get overflow?

Reply:
no, it is fine for this usecase, it would just happen if the diff between other and self is extremely large, just left it there for later